docs/release-notes/index.rst

```diff
@@ -3,8 +3,8 @@
 Release notes
 =============
 
-This page lists past release notes for Starcraft, summarising new features, bug
-fixes and backwards-incompatible changes in each version. It also contains the
+This page lists the notes for past releases of Starcraft, which summarize new features,
```

Comment:
```suggestion
This page lists the notes for past releases of Starcraft, which summarise new features,
```

Based on https://docs.ubuntu.com/styleguide/en/#spelling

Reply:
Snapcraft's docs have recently switched over to US English and I'm gearing up to make tasks for Charmcraft and Rockcraft. This is in anticipation of that.

Reply:
Would this change be better if I also adjust "behaviour" in the template's comments?

Reply:
Given that that's in opposition to the style guide, no :-)

Reply:
Switched it back: [977b775](https://github.com/canonical/starbase/pull/328/commits/977b77533625c3cb8dfaa385f46d4b46d23e75d8)